src/spikeinterface/extractors/neoextractors/neuralynx.py

```diff
@@ -55,13 +55,14 @@ class NeuralynxSortingExtractor(NeoBaseSortingExtractor):
 
     mode = "folder"
     NeoRawIOClass = "NeuralynxRawIO"
-    handle_spike_frame_directly = False
+    neo_returns_timestamps = True
+    need_t_start_from_signal_stream = True
     name = "neuralynx"
 
-    def __init__(self, folder_path, sampling_frequency=None):
+    def __init__(self, folder_path: str, sampling_frequency: float | None = None):
         neo_kwargs = self.map_to_neo_kwargs(folder_path)
         NeoBaseSortingExtractor.__init__(self, sampling_frequency=sampling_frequency, **neo_kwargs)
-        self._kwargs.update(dict(folder_path=str(folder_path)))
+        self._kwargs.update(dict(folder_path=str(folder_path), sampling_frequency=self._sampling_frequency))
```

Comment:
If we do that we can loose the t_start on the next reload from kwargs because setting frequency reset t_start.
Am I wrong ?

Reply:
Yes, that's a good point. Don't know how to solve this in a good way other than exposing either t_start of the stream_index.

Reply:
Good idea!